tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/DefaultListenerRegistry.java

```diff
@@ -35,7 +35,7 @@ public void addEventListener(EventListener listener) {
 	public void dispatch(Event event) {
 		Collection<EventListener> listeners;
 		synchronized (_lock) {
-			listeners = Collections.unmodifiableCollection(_listeners);
+			listeners = new ArrayList<>(_listeners);
```

Comment:
There has a `ConcurrentModificationException` (Take a glance inside `Collections.unmodifiableCollection`, it will copy the collection's reference)